docker-commands.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+if [[ "$1" == "build" ]]; then
+    if [[ "$2" == "app" ]]; then
+        echo "building app"
+        docker build -t materials-designer:centos -f dockerfiles/centos/Dockerfile .
+    elif [[ "$2" == "test" ]]; then
+        echo "building test"
+        docker build -t materials-designer:test -f dockerfiles/test/Dockerfile .
+    fi
+elif [[ "$1" == "run" ]]; then
+    if [[ "$2" == "app" ]]; then
+        echo "running app"
+        docker run -it -p 3001:3001 -d --name=centos materials-designer:centos
+    elif [[ "$2" == "test" ]]; then
+        echo "running test"
+        docker run -it --network=host --name=test materials-designer:test
+    fi
+elif [[ "$1" == "" ]]; then
```

Comment:
I'd swap the logic to print readme when no args is passed, and add "all" option instead to do what's on lines 20-25 below

Reply:
Done